ArduCopter/Log.cpp

```diff
@@ -76,6 +76,7 @@ void Copter::Log_Write_Attitude()
         logger.Write_PID(LOG_PIDP_MSG, attitude_control->get_rate_pitch_pid().get_pid_info());
         logger.Write_PID(LOG_PIDY_MSG, attitude_control->get_rate_yaw_pid().get_pid_info());
         logger.Write_PID(LOG_PIDA_MSG, pos_control->get_accel_z_pid().get_pid_info() );
+        logger.Write_PID(LOG_PIDN_MSG, pos_control->get_vel_xy_pid().get_pid_info_x() );
```

Comment:
I think we also need y
```suggestion
        logger.Write_PID(LOG_PIDN_MSG, pos_control->get_vel_xy_pid().get_pid_info_x() );
        logger.Write_PID(LOG_PIDN_MSG, pos_control->get_vel_xy_pid().get_pid_info_y() );
```

Reply:
also a new message name of course

Reply:
also unless there is a change to the pos control rate I think this will result in 40 logs of the same value? 400hz vs 10hz?

Reply:
I have made changes to address this. Randy will bring them in after addressing other issues here.